csrankings-c.csv

```diff
@@ -332,6 +332,7 @@ Changhee Jung,Purdue University,https://www.cs.purdue.edu/homes/chjung,_sIzYHAAA
 Changho Suh,KAIST,https://sites.google.com/site/changhosuh,B1guGw8AAAAJ
 Changhun Kim,Korea University,http://kucg.korea.ac.kr/new/about/people.php,NOSCHOLARPAGE
 Changick Kim,KAIST,https://cilabs.kaist.ac.kr/members/professor,ABH_2lcAAAAJ
+Changjian Li,University of Edinburgh,https://enigma-li.github.io/,aCgyQsQAAAAJ

```

Comment:
Disambiguation entry (https://dblp.org/pid/121/6228.html). Please reach out to DBLP per instructions.

> If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers ([dblp@dagstuhl.de](mailto:dblp@dagstuhl.de)) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.